applications/compareRestart.hpp

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
i guess exposing the UNSTReader might make sense, but in that case it should go in a separate compilation unit, and only the class definition be exposed. you can then include and use this class from the application. that makes sense if you want to build more than one application using the class.